### ISSUE TITLE
Response timeout when HTTPS enabled

### ISSUE THREAD

**Issue by bedla** [tomakehurst/wiremock#1190]
Hi,
I have strange behavior with Wiremock. I have tests for both HTTP and HTTPS endpoints, additional tests are with `withFixedDelay(..)` (to test read-time-out exception).
When I enable `dynamicHttpsPort()` second test starts to fail - when I comment-out that line test is passing. Please look at reporduction test https://github.com/bedla/wiremock-delay-reproduce .
Is it bug or feature? :] Or am I missing something.
I know that I can use other methods how to create instance of wiremock (this is basically because I use wiremock with junit5 and kotlin). The point is about creating it as you can see in my repository.
Thx,
Ivos

**Comment by bedla** [comment]
what I investigated:
1. `WireMockHandlerDispatchingServlet.ServletHttpResponder#delayIfRequired` somehow infer with second request
2. it is because sleep `delayMillis` value is high enough to accommodate 2x read-timeout (see `downloadContentAndMeasure` method in test)
2a. I can see blocked threads in YourKit monitor profiling window
3. when I tune `.jettyAcceptors(acceptorsCount)` and `.containerThreads(containerThreadsCount)` methods (I have 8 cores):
3a. `acceptorsCount = 3`, `containerThreadsCount = 10,11` I got `java.lang.IllegalStateException: Insufficient configured threads: required=11 < max=10 for QueuedThreadPool[qtp1688782916]@64a8c844{STARTED,8<=8<=10,i=3,r=1,q=0}[ReservedThreadExecutor@319988b0{s=0/1,p=0}]`
3b. `acceptorsCount = 3`, `containerThreadsCount = 12` I got usual `java.net.SocketTimeoutException: Read timed out`
3c. `acceptorsCount = 3`, `containerThreadsCount = 13+` test is green :]

Result: It looks like that Jetty internal pooling does not have enough acceptors (default is `2`) and threads to handle this situation/configuration. From my current understanding, one acceptor is for HTTP, one is for HTTPS (even not used in test) ... when I block/sleep one with "read-timeout test", no free available acceptor is ready for second "content test".
I dont know if Wiremock is somehow able to detect this situation or If it is sufficient to make a note in documentation.

Note: I have updated test case repo a bit.
